learning-ojs/en/journal-setup.md

```diff
@@ -110,11 +112,30 @@ The Website Settings allow you to configure how your journal's website looks and
 
 ### Appearance
 
-**Logo**: Use this space to upload a journal logo to the upper left corner of your reader interface.
+**Logo**: Use this space to upload a journal logo to the upper left corner of your reader interface. 
 
 **Page Footer**: This allows you to display text at the bottom of each page of your website.
 
-**Theme**: As new themes become available, you can enable them here.
+**Theme**: The theme determines the overall design or layout of your site. Several different theme options are available and you can try them out without affecting your site's content or configuration.
+
+First you can ensure that all available themes have been enabled on your site.
+1. Go to the Plugins tab under Website Settings
+2. Scroll to the bottom and find Theme Plugins
+3. Check the box next to each plugin to enable it
+
+You can also look for additional themes in the Plugin Gallery and install and enable those.
+
+Now that you have all available themes, return to the Appearance tab to try out different themes.
+1. Under Theme you will see a drop-down list of themes. Select one.
+2. Scroll to the bottom of the page and click Save
+3. The theme may have additional sub-themes or configuration options available. To make these appear, refresh the page in your browser.
```

Comment:
Just a heads-up that this will change in 3.2. They won't have to refresh the page to see the new theme options. :pray:

Reply:
That is great news!  We'll have to remember to change this when 3.2 is released.